extensions/amp-viewer-integration/CAPABILITIES.md

```diff
@@ -27,7 +27,8 @@ Viewers can communicate their supported "capabilities" to documents through the
 | `fragment`             | `fragment`                            | URL fragment support for the history API.                                       |
 | `handshakepoll`        | `handshake-poll`                      | Mobile web handshake.                                                           |
 | `iframeScroll`         |                                       | Viewer platform supports and configures scrolling on the AMP document's iframe. |
-| `navigateTo`           | `navigateTo`                          | Support for navigating to external URLs.                                        |
+| `interceptNavigation`  | `navigateTo`                          | Support for navigating to external URLs.                                        |
```

Comment:
This also need to be reflected in extensions/amp-viewer-integration/amp-doc-viewer-api.md

Reply:
Added this doc to amp-doc-viewer-api.md as well